Nodejs/Product/TargetsVsix/Microsoft.NodejsTools.targets

```diff
@@ -55,9 +53,18 @@
   </ItemGroup>
 
   <Target Name="CopyFilesToOutputDirectory"/>
+  
+  <Target Name="AfterClean">
+    <Exec Condition="'$(EnableTypeScript)' == 'true'" Command="&quot;$(Tsc)&quot; --build --clean" />
+  </Target>
+  
+  <Target Name="AfterCompile">
+    <Message Text="AfterCompile executed" />
+    <Exec Condition="'$(EnableTypeScript)' == 'true'" Command="&quot;$(Tsc)&quot; --build" />
```

Comment:
Having `--build ` flag enabled will force the user to have a `tsconfig.json` file in the project.